### PR TITLE
feat: [CDS-76235]: namespaced agent docs

### PR DESCRIPTION
**Pull request by mteodor**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by ashinsabu3 — DISMISSED** [review]

**Review by mankrit-singh — APPROVED** [review]

File: docs/continuous-delivery/gitops/use-gitops/install-a-harness-git-ops-agent.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ In **Name**, enter the name for the new Agent.
 
 In **Namespace**, enter the namespace where you want to install the Harness GitOps Agent. Typically, this is the target namespace for your deployment.
 
+If **Namespaced** is selected, the Agent will only be able to access resources in the namespace where it is installed. If **Cluster** is selected, Harness GitOps Agent will be installed without cluster-scoped permissions, additionally you can select **Skip Crds** to avoid collision if already installed.
```
Comment (meenaravichandran1) [inline docs/continuous-delivery/gitops/use-gitops/install-a-harness-git-ops-agent.md:90]: is SkipCrds an option in the UI or command?

Comment (manavjot-harness) [inline docs/continuous-delivery/gitops/use-gitops/install-a-harness-git-ops-agent.md:90]: If Cluster is selected? I dont think we have a selection for that?

Comment (mteodor) [reply docs/continuous-delivery/gitops/use-gitops/install-a-harness-git-ops-agent.md]: in the UI, you can observe on qa

Comment (mteodor) [reply docs/continuous-delivery/gitops/use-gitops/install-a-harness-git-ops-agent.md]: yep, a mistake, good catch

**Review by manavjot-harness — DISMISSED** [review]

**Review by ashinsabu3 — APPROVED** [review]

**Pull request merged**